docs/doc/11-integrations/30-access-tool/00-mysql.md

```diff
@@ -31,14 +35,56 @@ mysql -h127.0.0.1 -uroot -P3307
 
 ## Built-in Accounts
 
-There are four built-in accounts in Databend that can be accessed from a local machine without requiring a password. These accounts are immutable and possess the highest level of privileges, meaning their settings cannot be altered or removed. Additionally, it is not possible to change or update their passwords.
+Databend comes with four preconfigured root users that can be accessed from a local machine without requiring a password:
 
 - `root@127.0.0.1`
 - `root@localhost`
```

Comment:
There are not default builtin users, all builtin users should be configured in `query.toml.`

The following four users should be deleted in docs.